src/DynamoApplications/AssemblyConflictChecker.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using DynamoApplications.Properties;
+
+namespace Dynamo.Applications
+{
+    /// <summary>
+    /// Checks for incompatibilities between loaded assemblies.
+    /// </summary>
+    public class AssemblyConflictChecker : IDisposable
+    {
+        /// <summary>
+        /// List of assembly names to ingnore when checking for incompatibilities.
+        /// </summary>
+        public IEnumerable<string> AssemblyNamesToIgnore = new List<string>();
```

Comment:
can we make this a property?
see this quote from MSDN:
```
Generally, you should use fields only for variables that have private or protected accessibility. Data that your class exposes to client code should be provided through methods, properties, and indexers.
```
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/fields